src/components/pill/pill.spec.js

```diff
@@ -362,16 +361,133 @@ describe('Pill', () => {
       }, TestRenderer.create).toJSON();
       assertStyleMatch({
         borderRadius: '12px',
-        fontSize: '14px',
+        fontSize: '10px',
         fontWeight: '600',
         position: 'relative',
         top: '-1px',
-        padding: '2px 8px 2px 8px',
+        padding: '4px 8px 2px 8px',
         margin: '0px 8px 16px 0px'
       }, wrapper);
     });
   });
 
+  describe('classic theme', () => {
+    const renderBase = (props, renderer = mount) => {
+      return renderer(
+        <ThemeProvider theme={ classicTheme }>
```

Comment:
This should be called renderClassic.

I'm also not sure any of these theme-specific methods are actually required, I would have thought you could just do:

```
const render = (props, renderer = mount) => {
  return renderer(
    <Pill { ...props } />
  )
}
```
and have `theme: classicTheme` or `theme: `baseTheme` in the props you pass. (Might not even need baseTheme for that one, as it may already be the default).

Then pull the function out a level so all your describe blocks can access it, and you'll save on a lot of repetition.

I would also check if the methods are ever currently being called _without_ passing `TestRenderer.create` as the renderer param - if they aren't, you can remove the param entirely and hard-code that, and if they are but only a few, then maybe worth looking at making `TestRenderer.create` the default param instead of `mount`.

Reply:
stuck with one render method, indeed there was no need to have many.